app/config/config.yml

```diff
@@ -7,6 +7,7 @@ imports:
 # http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
     locale: en
+    swp_updater.version.class: SWP\UpdaterBundle\Version\Version
```

Comment:
ahhh... yeah, but then example should not to be committed (in our case we need here our project version class) - in AppBundle (please also rename AppBundle to SWP\WebRendererBundle)